aptos-move/move-examples/fungible_asset/Move.toml

```diff
@@ -1,10 +1,9 @@
 [package]
-name = "FungibleAsset"
```

Comment:
There is a related issue I opened a few weeks ago, making examples downloadable: https://github.com/aptos-labs/aptos-core/issues/7081. 

In any case, the one or the other way, we need to do something here, because people continue to run into issues using examples at head with older CLIs.